corehq/apps/api/resources/v0_5.py

```diff
@@ -799,15 +799,20 @@ def dispatch_list(self, request, **kwargs):
                 raise
 
     def obj_get_list(self, bundle, **kwargs):
-        return self.get_object_list(bundle.request)
+        feature_flag = bundle.request.GET.get("feature_flag")
+        if feature_flag and feature_flag not in toggles.all_toggles_slug():
+            raise BadRequest("{} is not a valid feature flag".format(feature_flag))
+        return self.get_object_list(bundle.request, feature_flag)
 
-    def get_object_list(self, request):
+    def get_object_list(self, request, feature_flag=None):
         couch_user = CouchUser.from_django_user(request.user)
         results = []
         for domain in couch_user.get_domains():
             if not domain_has_privilege(domain, privileges.ZAPIER_INTEGRATION):
                 continue
             domain_object = Domain.get_by_name(domain)
+            if feature_flag and feature_flag not in toggles.toggles_dict(domain=domain).keys():
```

Comment:
Do you need to check if the FF is enabled here? Also, a user may have a flag enabled, but not a domain - what is the outcome there?

Reply:
> What is the expected behavior if flag enabled for user but not for domain?

Then it is enabled. i.e. It is an "or" operation: enabled = enabled_for_domain or enabled_for_user

> We may need to pass user also in the call along with domain accordingly.

:+1: